test/Microsoft.NET.Build.Tests/GivenThatWeWantToBuildALibrary.cs

```diff
@@ -577,5 +577,15 @@ public void It_passes_ridless_target_to_compiler()
                 .Should()
                 .Pass();
         }
+
+        [Fact]
+        public void It_restores_only_ridless_tfm()
```

Comment:
`build -FullMSBuild` did not fail. 

Does "Full MSBuild" mean the sdk and targets under `%VSINSTALLDIR%` or under `.dotnet_cli`. I assume its under `.dotnet_cli`. In this case, these are the same "ImportAfter" targets used by the CLI i.e. they have already removed Microsoft.NuGet.targets (see `.dotnet_cli\sdk\1.0.0-rc4-004771\15.0\Microsoft.Common.targets\ImportAfter\Microsoft.NuGet.ImportAfter.targets`)

I inspected the project.assets.json generated for the test under `bin\Debug\Tests`, and it only contains one (ridless) targets entry, so the test is correctly passing, but would fail if we were using the MSBuild under `%VSINSTALLDIR%`.